roles/cifmw_cephadm/templates/ceph_rgw.yml.j2

```diff
@@ -29,11 +29,11 @@ spec:
     backend_service: rgw.rgw
     frontend_port: 8080
     monitor_port: 8999
-    virtual_ip: {{ cifmw_cephadm_rgw_vip }}
     virtual_interface_networks:
     - {{ cifmw_cephadm_rgw_network }}
-  {% if rgw_frontend_cert is defined %}
+    virtual_ip: {{ cifmw_cephadm_rgw_vip }}
+{% if rgw_frontend_cert is defined %}
```

Comment:
The `rgw_frontend_cert` needs to contain not just the cert but the cert and the key.

```yaml
  ssl_cert: |
    -----BEGIN CERTIFICATE-----
    M...
     rPXmtAhwVQ==
    -----END CERTIFICATE-----
    -----BEGIN RSA PRIVATE KEY-----
    MIIEp ...
```